torch/nn/utils/rnn.py

```diff
@@ -208,12 +169,36 @@ def pad_packed_sequence(input, *args, **kwargs):
         containing the list of lengths of each sequence in the batch.
 
     """
-    import torch
```

Comment:
to be clear, achieving the cleanup in this area is effectively the whole point of the diff (to be in line with what Edward proposed in https://github.com/pytorch/pytorch/pull/5420 and https://github.com/pytorch/pytorch/issues/5460)